backend/app/app/api/api_v1/endpoints/enroll.py

```diff
@@ -2,10 +2,12 @@
 
 import psycopg2.errors
 from fastapi import APIRouter, Depends, responses, status
+from starlette.status import HTTP_401_UNAUTHORIZED
```

Comment:
already imported `fastapi.status` provides exactly the same functions and constants. Consider using `status.HTTP_401_UNAUTHORIZED`